common/src/indexer_service/http/config.rs

```diff
@@ -42,6 +42,13 @@ pub struct IndexerServiceConfig {
     pub escrow_subgraph: SubgraphConfig,
     pub graph_network: GraphNetworkConfig,
     pub tap: TapConfig,
+    pub dips: IndexerDipsConfig,
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct IndexerDipsConfig {
+    pub expected_payee: String,
+    pub allowed_payers: Vec<String>,
```

Comment:
Let's use the `thegraph_core::Address` type here, as we'll provide Ethereum addresses. This way, we can validate at deserialization time.